go.mod

```diff
@@ -121,10 +124,5 @@ require (
 // replace github.com/hashicorp/horizon => ../horizon
 // replace github.com/hashicorp/waypoint-plugin-sdk => ../waypoint-plugin-sdk
 
-replace (
-	// v0.3.11 panics for some reason on our tests
-	github.com/imdario/mergo => github.com/imdario/mergo v0.3.9
-
-	// https://github.com/ory/dockertest/issues/208
-	golang.org/x/sys => golang.org/x/sys v0.0.0-20200826173525-f9321e4c35a6
-)
+// v0.3.11 panics for some reason on our tests
+replace github.com/imdario/mergo => github.com/imdario/mergo v0.3.9
```

Comment:
So the sys stuff is just fixed now?

Reply:
Yup! everything compiles fine now, the offending package has been fixed.

Reply:
YES